installer/scripts/omsadmin.sh

```diff
@@ -346,6 +362,28 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
The scope of this task was to control the number of processes under the omsagent user specifically.
The hybrid worker should be able to set a different limit for the nxautomation user if they need to in the future.